verify.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Verify TextFSM template
+  hosts: localhost
+  gather_facts: False
+
+  vars_prompt:
+    - name: template_name
+      prompt: "Enter the name of the template to verify"
+      default: "cisco_show_vlan"
+      private: no
+
+  roles:
```

Comment:
I'm thinking you can combine all of this into a ONE role called test_template.  It could in theory, call other roles, but I'm fine if the test_template role does all of the tasks from test_env_setup, debug (again, would rather not have a role, that just calls debug at this point), then doesn't have to call a ntc role, but rather just the ntc_show_command, and then finishes with a task that calls compare_dict.

I'm open, but I want to keep this simple as we really want to make it easy for network engineers to use, etc.

One more thing- can you call this playbook test-template-playbook.yml instead of verify.yml?



Thoughts?